### PR TITLE
Add Meeting Test Cases

### PR DESCRIPTION
**Pull request by MerlinLim**
Create Add Command and Model Meeting test cases.

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/meeting/CommandTestUtil.java [hunk 1]
```diff
@@ -52,12 +52,12 @@
     public static final String LOCATION_DESC_A = " " + PREFIX_LOCATION + VALID_LOCATION_A;
     public static final String LOCATION_DESC_B = " " + PREFIX_LOCATION + VALID_LOCATION_B;
 
-    public static final String INVALID_TITLE_DESC = " " + PREFIX_TITLE + "Meeting&"; // '&' not allowed in names
+    public static final String INVALID_TITLE_DESC = " " + PREFIX_TITLE + ""; // '&' not allowed in names
```
Comment (claraadora) [inline src/test/java/seedu/address/logic/commands/meeting/CommandTestUtil.java:55]: I think the comment here is outdated

File: src/test/java/seedu/address/logic/commands/meeting/CommandTestUtil.java [hunk 2]
```diff
@@ -52,12 +52,12 @@
...
     public static final String INVALID_DESCRIPTION_DESC = " " + PREFIX_DESCRIPTION + ""; // 'a' not allowed in phones
-    public static final String INVALID_FROM_DESC = " " + PREFIX_FROM + "bob!yahoo"; // missing '@' symbol
-    public static final String INVALID_TO_DESC = " " + PREFIX_TO; // empty string not allowed for addresses
-    public static final String INVALID_CONTACTS_DESC = " " + PREFIX_CONTACTS + "hubby*"; // '*' not allowed in tags
-    public static final String INVALID_LOCATION_DESC = " " + PREFIX_LOCATION + "Home*"; // 'friend' not allowed in roles
+    public static final String INVALID_FROM_DESC = " " + PREFIX_FROM + "3-03-2020 09:30"; // missing '0' digit
+    public static final String INVALID_TO_DESC = " " + PREFIX_TO + "03-03-2020 0930"; // missing colon
+    public static final String INVALID_CONTACTS_DESC = " " + PREFIX_CONTACTS + "1 2 3"; // no commas
+    public static final String INVALID_LOCATION_DESC = " " + PREFIX_LOCATION + ""; // 'friend' not allowed in roles
```
Comment (claraadora) [inline src/test/java/seedu/address/logic/commands/meeting/CommandTestUtil.java:60]: I think this as well?

File: src/test/java/seedu/address/model/meeting/meeting/LocationTest.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import org.junit.jupiter.api.Test;
+
+class LocationTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Location((String) null));
+    }
+
+    @Test
```
Comment (claraadora) [inline src/test/java/seedu/address/model/meeting/meeting/LocationTest.java:16]: Is this supposed to be constructor_invalidLocation?

File: src/test/java/seedu/address/model/meeting/meeting/UniqueMeetingListTest.java [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    @Test
+    public void contains_personNotInList_returnsFalse() {
+        assertFalse(uniqueMeetingList.contains(MEETING_A));
+    }
+
+    @Test
+    public void contains_personInList_returnsTrue() {
+        uniqueMeetingList.add(MEETING_A);
+        assertTrue(uniqueMeetingList.contains(MEETING_A));
+    }
```
Comment (claraadora) [inline src/test/java/seedu/address/model/meeting/meeting/UniqueMeetingListTest.java:40]: I think we can change the method names to meeting here?

File: src/test/java/seedu/address/model/meeting/meeting/UniqueMeetingListTest.java [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+
+    //    @Test
+    //    public void contains_MeetingWithSameIdentityFieldsInList_returnsTrue() {
+    //        uniqueMeetingList.add(MEETING_A);
+    //        Meeting editedMeetingA = new MeetingBuilder(MEETING_A).withDescription(VALID_DESCRIPTION_B)
+    //                .build();
+    //        assertTrue(uniqueMeetingList.contains(editedMeetingA));
+    //    }
+
```
Comment (claraadora) [inline src/test/java/seedu/address/model/meeting/meeting/UniqueMeetingListTest.java:49]: Should we delete this comment or add a TODO comment?

**Review by claraadora — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/address/model/meeting/meeting/UniqueMeetingListTest.java (hunk 5 above)
Comment (MerlinLim) [reply src/test/java/seedu/address/model/meeting/meeting/UniqueMeetingListTest.java:49]: Maybe add a TODO comment

Comment (claraadora) [reply src/test/java/seedu/address/model/meeting/meeting/UniqueMeetingListTest.java:49]: Okayy

> Maybe add a TODO comment

**Review by claraadora — APPROVED** [review]

**Pull request merged**